src/components/v1/ProjectPayoutMods.tsx

```diff
@@ -437,15 +436,15 @@ export default function ProjectPayoutMods({
           }}
           block
         >
-          <Trans>Add a split</Trans>
+          <Trans>Add a payout</Trans>
         </Button>
       </Space>
 
       <Modal
-        title={modalMode === 'Edit' ? t`Edit existing split` : t`Add a split`}
+        title={modalMode === 'Edit' ? t`Edit existing split` : t`Add a payout`}
         visible={editingModIndex !== undefined}
         onOk={setReceiver}
-        okText={modalMode === 'Edit' ? t`Save split` : t`Add split`}
+        okText={modalMode === 'Edit' ? t`Save split` : t`Add payout`}
```

Comment:
**suggestion** should this be `Save payout`?

```suggestion
        okText={modalMode === 'Edit' ? t`Save payout` : t`Add payout`}
```

Reply:
Should every reference to `split` in this file be changed to `payout`?

Reply:
Thought I'd just keep all v1 wording the same. Saw I missed changing some back- they're all 'payout' now